docs/src/pages/components/buttons/buttons.md

```diff
@@ -180,3 +180,30 @@ However:
 ```
 
 This has the advantage of supporting any element, for instance, a link `<a>` element.
+
+## Unstyled button
+
+The button also comes with an unstyled version. It's ideal for doing heavy customizations and minimizing bundle size.
+
+### Unstyled component
+
+```jsx
+import ButtonUnstyled from '@material-ui/unstyled/ButtonUnstyled';
+```
+
+By default, the `ButtonUnstyled` renders a native `button` element.
+You are free to override this by setting the `component` or `components.Root` prop.
+If a non-interactive element (such as a span) is provided this way, the `ButtonUnstyled` will take care of adding accessibility attributes.
+
+{{"demo": "pages/components/buttons/UnstyledButtons.js"}}
+
+### useButton hook
+
+```jsx
```

Comment:
I'm not sure why, we didn't use jsx when they were no JSX on the page. Maybe we should always use jsx or tsx for simplicity.

````suggestion
```js
````